bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/unit/BinaryPrefix.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.core.library.unit;
+
+import java.math.BigInteger;
+
+import javax.measure.Quantity;
+import javax.measure.Unit;
+import javax.measure.UnitConverter;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+import tec.uom.lib.common.function.SymbolSupplier;
+import tec.uom.lib.common.function.UnitConverterSupplier;
+import tec.uom.se.function.RationalConverter;
+
+/**
+ * The binary prefixes used to derive units by specific powers of 2.
+ *
+ * @author Gaël L'hopital - initial contribution and API
+ *
+ */
+@NonNullByDefault
+public enum BinaryPrefix implements SymbolSupplier, UnitConverterSupplier {
+    YOBI("Yi", new RationalConverter(BigInteger.valueOf(2).pow(80), BigInteger.ONE)),
+    ZEBI("Zi", new RationalConverter(BigInteger.valueOf(2).pow(70), BigInteger.ONE)),
+    EXBI("Ei", new RationalConverter(BigInteger.valueOf(2).pow(60), BigInteger.ONE)),
+    PEBI("Pi", new RationalConverter(BigInteger.valueOf(2).pow(50), BigInteger.ONE)),
+    TEBI("Ti", new RationalConverter(BigInteger.valueOf(2).pow(40), BigInteger.ONE)),
+    GIBI("Gi", new RationalConverter(BigInteger.valueOf(2).pow(30), BigInteger.ONE)),
+    MEBI("Mi", new RationalConverter(BigInteger.valueOf(2).pow(20), BigInteger.ONE)),
+    KIBI("Ki", new RationalConverter(BigInteger.valueOf(2).pow(10), BigInteger.ONE));
+
+    /**
+     * The symbol of this prefix, as returned by {@link #getSymbol}.
+     *
+     * @serial
+     * @see #getSymbol()
+     */
+    private final String symbol;
+
+    /**
+     * The <code>UnitConverter</code> of this prefix, as returned by {@link #getConverter}.
+     *
+     * @serial
+     * @see #getConverter()
+     * @see {@link UnitConverter}
+     */
+    private final UnitConverter converter;
+
+    /**
+     * Creates a new prefix.
+     *
+     * @param symbol
```

Comment:
This does not look like the right formatting. Please check your settings. Perhaps this https://github.com/eclipse/smarthome/issues/6424#issuecomment-471728524 will help.